src/main/scala/com/sksamuel/scapegoat/sbt/ScapegoatSbtPlugin.scala

```diff
@@ -43,6 +43,8 @@ object ScapegoatSbtPlugin extends AutoPlugin {
       Defaults.compileSettings ++
       Seq(
         sources := (sources in Compile).value,
```

Comment:
@gregsymons, do you know why we need to explicitly set `sources` here? I see that you added this line in #24 when introducing the separate `scapegoat` task.

Based on my limited understanding of SBT configurations, I think that the intent behind the `val Scapegoat = config("scapegoat") extend Compile` further up in this file was that our `Scapegoat` configuration would inherit all of the `Compile` configuration's settings for settings which weren't explicitly defined. If that was the case, though, I guess we wouldn't need to explicitly define `sources` here. This kind of explains why I needed to also add explicit lines for `managedClasspath` and `unmanagedClasspath` in order to fix compilation with `provided` dependencies, but I wonder if there's a more principled fix that will implicitly inherit all of the undefined settings from `Compile` rather than forcing us to explicitly copy them here (I'm worried about leaving out other settings in a way that's confusing to users, e.g. leading to different compilation results).